src/braket/analog_hamiltonian_simulator/rydberg/validators/atom_arrangement.py

```diff
@@ -0,0 +1,113 @@
+import warnings
+from decimal import Decimal
+from typing import Dict, Tuple
+
+import numpy as np
+from braket.ir.ahs.atom_arrangement import AtomArrangement
+from pydantic.class_validators import root_validator
+
+from braket.analog_hamiltonian_simulator.rydberg.validators.capabilities_constants import (
+    CapabilitiesConstants,
+)
+
+
+def euclidean_distance(site_1: Tuple[Decimal, Decimal], site_2: Tuple[Decimal, Decimal]) -> Decimal:
+    # Compute the Euclidean distance between two sets of 2-D points, (x1, y1) and (x2, y2)
+
+    return np.linalg.norm(np.array(site_1) - np.array(site_2))
```

Comment:
Can be made private. If it _needs_ to be public, then it needs a proper docstring.